### DISCUSSION TITLE
Meeting Notes for GSoC (8/1/2024)

### DISCUSSION THREAD

**Discussion by JackyZzZz** [theochem/Selector#233] [General]
# Meeting Notes

**Date:** 8/1/2024
**Participants:** @FanwangM   @JackyZzZz

## Meeting Overview

1. **Workflow Issues**
   - Finally we were able to pin the exact reason why the workflow keeps failing. Github doesn't allow outside pull request to access repo's environment secrets, causing our workflow to fail. We did a simple experiment today since Fanwang have write access to the repo. The workflow works fine then. 
 
2. **Matrix Converter**
    - Fanwang shared some resources about chemical formats such as `smiles` and `sdf`, will work on it next.
      - https://www.rdkit.org/docs/GettingStartedInPython.html
      - https://chem.libretexts.org/Courses/Intercollegiate_Courses/Cheminformatics/02%3A_Representing_Small_Molecules_on_Computers/2.05%3A_Structural_Data_Files
      - https://pubchem.ncbi.nlm.nih.gov/

3. **Review of PRs**
- Fanwang will coordinate to help me get write access to the repo so I can make all the workflow running correctly. However, in order to minimize the effect of web server on scenarios such as publication. I will notify Fanwang before raising the pull request so that he will open the PR and review it ASAP.

## Next Steps

**To-Do Before Next Meeting:**
- Understanding and working on the Matrix Converter interface.
- Finish resolving any changes needed for the PRs.

**Comment by FanwangM** [comment]
Repo access granted. @JackyZzZz
